donjo-app/controllers/Laporan.php

```diff
@@ -55,7 +55,7 @@ public function index($lap = 0, $p = 1, $o = 0)
 
 		$data['bulan'] = $data['bulanku'];
 		$data['tahun'] = $data['tahunku'];
-		$data['config'] = $this->laporan_bulanan_model->configku();
+		$data['config'] = $this->$this->config_model->get_data();
```

Comment:
Duplicate keyword `$this`

Reply:
Duplicate keyword masih ada di beberapa bagian. Silakan dicek kembali

Reply:
Yg mana pak @dikisiswanto ??? Tolong dibantu

Reply:
`$this->$this->config_model->get_data();` seharusnya `$this->config_model->get_data();`

Reply:
Sudah pak @dikisiswanto , search pakai vscode dapat 4 biji